### ISSUE TITLE
postgresql 10 breaks sequence handling

### ISSUE THREAD

**Issue by darix** [mbleigh/seed-fu#122]
```
seed-fu-2.3.6/lib/seed-fu/seeder.rb
95:            SELECT setval('#{sequence}', (SELECT GREATEST(MAX(#{quoted_id})+(SELECT increment_by FROM #{sequence}), (SELECT min_value FROM #{sequence})) FROM #{@model_class.quoted_table_name}), false)
```

This no longer works in postgresql 10. I think it is related to:

```
Move sequences' metadata fields into a new pg_sequence system catalog (Peter Eisentraut)

A sequence relation now stores only the fields that can be modified by nextval(), that is last_value, log_cnt, and is_called. Other sequence properties, such as the starting value and increment, are kept in a corresponding row of the pg_sequence catalog. ALTER SEQUENCE updates are now fully transactional, implying that the sequence is locked until commit. The nextval() and setval() functions remain nontransactional.

The main incompatibility introduced by this change is that selecting from a sequence relation now returns only the three fields named above. To obtain the sequence's other properties, applications must look into pg_sequence. The new system view pg_sequences can also be used for this purpose; it provides column names that are more compatible with existing code.

The output of psql's \d command for a sequence has been redesigned, too.
```

https://www.postgresql.org/docs/10/static/release-10.html

In worst a check on PG::Conn#server_version could be used.

**Comment by darix** [comment]
https://meta.discourse.org/t/discourse-appears-to-be-broken-with-postgres-10/71723

**Comment by radar** [comment]
It's likely that seed-fu would need to make a change like this: 
http://github.com/jeremyevans/sequel/commit/8370980b12f59d5439344d4de89d4552f05a3d13

**Comment by joshuawscott** [comment]
I think this would be fixed by https://github.com/mbleigh/seed-fu/pull/121 ?

**Comment by aserafin** [comment]
I'm closing this since #121 has just been merged and v2.3.7 is pushed to rubygems
